CMakeLists.txt

```
@@ -14,21 +14,37 @@
 cmake_minimum_required(VERSION 3.10)
 project(parthenon LANGUAGES C CXX)
 
-# Options
 include(CTest)
+
+# Compile time constants
+set(PARTHENON_NGHOST 2 CACHE STRING "Number of ghost cells") 
```

Is now the time to just make this an option?
```suggestion
option(PARTHENON_NGHOST "Number of ghost cells" 2)
```

---

What's the exact difference (asking out of curiosity and not because I think we should use one or the other)? I tried google but it seemed that it mostly made a difference for boolean variables (as they'd allow dependencies for `option` but not for `set`)?

---

The problem with option is that it is boolean, so it is only supposed to turn ghosts on or off.  We shouldn't use an option to set a specific value.  Hence not an option.